test/RoadRegistry.Tests/TestStartup.cs

```diff
@@ -39,6 +40,11 @@ public void ConfigureHost(IHostBuilder hostBuilder)
         var availableModuleAssemblyCollection = DetermineAvailableAssemblyCollection();
 
         hostBuilder
+            .ConfigureAppConfiguration((hostContext, configurationBuilder) =>
+            {
+                configurationBuilder
+                    .AddJsonFile("appsettings.test.json", optional: true, reloadOnChange: true);
```

Comment:
why not simply use appsettings.json? it's a test.

Reply:
Done